src/main/resources/application.yaml

```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api}
```

Comment:
dont u need rd_user_profile_api to run the functional tests?

Reply:
Yes you're correct, added it back to authorised list here: https://github.com/hmcts/rd-user-profile-api/pull/581/commits/29c21fed3716eadddcc2548eab34f3e41323e0be